docs/run-random-beacon.adoc

```diff
@@ -357,6 +357,8 @@ Once you've got your KEEP token grant you can manage it with our https://dashboa
 [small]*`"/dns4/bootstrap-2.test.keep.network/tcp/3919/ipfs/16Uiu2HAmNNuCp45z5bgB8KiTHv1vHTNAVbBgxxtTFGAndageo9Dp"`*
 [small]*`"/dns4/bootstrap-3.test.keep.network/tcp/3919/ipfs/16Uiu2HAm8KJX32kr3eYUhDuzwTucSfAfspnjnXNf9veVhB12t6Vf"`*
 [small]*`"/dns4/bootstrap-4.test.keep.network/tcp/3919/ipfs/16Uiu2HAkxRTeySEWZfW9C83GPFpQUXvrygmZryCN6DL4piZrbAv4"`*
+[small]*`"/dns4/bootstrap-1.core.keep.test.boar.network/tcp/3001/ipfs/16Uiu2HAkuTUKNh6HkfvWBEkftZbqZHPHi3Kak5ZUygAxvsdQ2UgG"`*
+[small]*`"/dns4/bootstrap-2.core.keep.test.boar.network/tcp/3001/ipfs/16Uiu2HAmQirGruZBvtbLHr5SDebsYGcq6Djw7ijF3gnkqsdQs3wK"`*
```

Comment:
If you render the document, the formatting seems to be broken:

![image](https://user-images.githubusercontent.com/4712360/80045640-5fb2b000-8508-11ea-9d58-1f20a7ab2798.png)

Reply:
I replaced the previous formatting (small + bold) in 1310396.

@sthompson22 I'm open to update this part to any format you would like to see here.